introduction.md

```diff
@@ -1,2 +1,16 @@
 Hi! My name is Thiago Medeiros.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?